game/utilities/check_nikos_hack.rpy

```diff
@@ -15,24 +15,22 @@ label check_nikos_hack:
         import sys
 
         #nikos_hack.main() #no problems here. It has to do with mock , new_callable = StringIO.StringIO
-        while True:
-            try:
-                nIKO_WUZ_HERE_LOL.main()
-                reload(nIKO_WUZ_HERE_LOL)
-                with mock.patch('sys.stdout', new_callable = StringIO.StringIO) as mock_stdout:
-                    nIKO_WUZ_HERE_LOL.main()
-                    if mock_stdout.getvalue() == 'Niko wuz here\n':
-                        renpy.say("Clara", "I am still locked out of my computer!")
-                        renpy.say("Clara", "I wonder where {a=hint1}Niko{/a} might have hidden his hack")
-                    else:
-                        break
-            except Exception as e:
-                renpy.say("Clara","Error with nIKO_WUZ_HERE_LOL.py")
-                renpy.say("Clara",str(e))
-                #renpy.say("Clara","Press enter to restart")
-                continue
 
-        del nIKO_WUZ_HERE_LOL
+        print_value = ""
```

Comment:
Hey Aidan! Thanks for this great bug fix. Can you leave a 1 line comment explaining why we are storing and printing this value.